src/analysis/modals/ComputeModal/GcpComputeModal/GcpComputeModal.js

```diff
@@ -667,6 +667,7 @@ export const GcpComputeModalBase = ({
       ..._.mapKeys((key) => `desiredRuntime_${key}`, desiredRuntime),
       desiredRuntime_exists: !!desiredRuntime,
       desiredRuntime_cpus: desiredRuntime ? desiredRuntimeCpus : undefined,
+      desiredRuntime_gpuEnabled: desiredRuntime ? desiredRuntime.gpuConfig?.numOfGpus > 0 : undefined,
```

Comment:
![Screenshot 2024-03-05 at 10 04 29 AM](https://github.com/DataBiosphere/terra-ui/assets/13465644/539de041-b88f-4b2f-81c3-47df902ba90b)

Would it make sense to just track whether the GPU box is enabled? Making sure that the GPU config exist, with more than 0 GPU also makes sense but I was wondering if we could simplify